src/components/DropArea.jsx

```diff
@@ -71,11 +84,14 @@ function DropArea({ defaultItems, theme, taskId }) {
                 taskId: taskId
               })
             }
-            onDrop={() => handleItemDrop(value)}>
+            onDrop={() => handleItemDrop(value)}
+            onDragOver={() => setShowLineStrong(false)}
+            onDragLeave={() => setShowLineStrong(true)}>
             <Component {...props} />
           </DraggableItem>
         );
       })}
+      {showLineStrong && showLine && <Line />}
```

Comment:
I think it would be easier to use one state with 3 values instead of 2 flag states

for example:
```js
const HOVER_STATE = {
    NONE: 'drag-hover-none',
    ITEM: 'drag-hover-item',
    AREA: 'drag-hover-area'
}
const [hover, setHover] = useState(HOVER_STATE.NONE);
```